test/1.2/events/T_after_chk.dml

```diff
@@ -34,9 +42,9 @@ attribute alttest {
     parameter configuration = "pseudo";
     method set(value) {
         log "info": "running alttest";
-        after (0.1) call $m1;
-        after (0.1) call $p[1][1].m1;
-        after (0.1) call $m2(1);
-        after (0.1) call $p[1][1].m2(4);
+        call $post_m1;
+        call $p[1][1].post_m1;
+        call $post_m2(1);
+        call $p[1][1].post_m2(4);
```

Comment:
I move these after calls to a method of the template to more thoroughly test checkpointing of domains (with after calls directly in `set` they would all be associated with `alttest`).

Reply:
Instead of simply covering domain checkpointing logic, these tests have been moved to under 1.4 and actually test that domains are checkpointed properly.

This leaves us with the question, what do we do about `T_after_chk_dup`? Its existence was already questionable. Do we remove it?